assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. By default, "this" binds to the Window. That means the value of "this" will be the Window Object.
+* 2. Implicit binding means "this" will be whatever called it. So, with example.this, "this" becomes "example".
+* 3. When using a constructor function, "this" becomes the name of the function created.
```

Comment:
```suggestion
* 3. When using a constructor function, "this" becomes the name of the object created.
```
constructor functions return a new object